<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN">
<html>
<head>

  


  
  <title>PAZAR Database Project Outline</title>
  <meta name="description" content="An open-access system for the collection and dissemination of regulatory sequence annotation">


  
  <meta name="keywords" content="database, regulatory sequences, annotation, pazar">


  
  <meta http-equiv="CONTENT-TYPE" content="text/html; charset=utf-8">
</head>


<body style="background-color: rgb(255, 255, 255);">


<center>
<table width="600">


  <tbody>


    <tr>


      <td width="600">
      
      <center>
      
      <p><b><i><span style="font-size: 20pt;">PAZAR</span></i></b><b><span style="font-size: 14pt;"> </span></b><b><span style="font-size: 20pt;"><i>-</i>
Project Outline</span></b></p>


      </center>


      
      <hr>
      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><b><span style="font-size: 34pt;"></span></b></p>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><b><span style="font-size: 14pt;">Introduction:</span></b></p>


      
      <p class="MsoNormal" style="text-align: justify;">Algorithms
and software for the analysis of transcription regulating
sequences have proliferated. Methods based on phylogenetic
footprinting, combinatorial interactions between transcription factors
and genomics data appear regularly. Unfortunately much of the research
in the field
continues to re-analyze the same data. In order to advance this active
field, it is critical to gain unrestricted access to a broader range of
reference data. Such&nbsp;collections would be useful both for the
measurement of software performance and, most importantly, as fertile
ground for investigation. Small collections appear periodically in
independent databases, often as by-products of algorithm development
projects. For instance, the <a target="_blank" href="http://jaspar.cgb.ki.se/cgi-bin/jaspar_db.pl">JASPAR</a>
database of binding profiles emerged
in this manner.</p>


      
      <p class="MsoNormal" style="text-align: justify;">PAZAR
is a
software framework for the
construction and maintenance of regulatory sequence data annotations; a
framework which
allows multiple boutique databases to function independently within a
larger system
(or information mall). &nbsp;Our goal is to be the <span style="font-weight: bold;">public
repository for regulatory data</span>.</p>


      
      <p class="MsoNormal" style="text-align: justify;"><a href="PAZARposter.pdf">Download PAZAR's poster</a></p>


      
      <p class="MsoNormal" style="text-align: justify;"></p>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><b><span style="font-size: 14pt;"></span></b></p>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><b><span style="font-size: 14pt;"></span></b></p>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><b><span style="font-size: 14pt;">PAZAR's
principles:</span></b></p>


      
      <p class="MsoNormal" style="text-align: justify;">(1)
to be
OPEN-ACCESS and OPEN-SOURCE, providing a completely transparent
development and data compilation. In this regard, the PAZAR project is
now hosted by <a target="_blank" href="http://sourceforge.net/projects/pazar">sourceforge.net</a>,
where everyone can go and browse the <a href="http://cvs.sourceforge.net/viewcvs.py/pazar/" target="_blank">CVS
repository</a>. A mailing list <a href="http://lists.sourceforge.net/lists/listinfo/pazar-news" target="_blank">'News and Views'</a> is also
available in which every major development will be posted.</p>


      
      <p class="MsoNormal" style="text-align: justify;">(2)
to
function as a boutique system where curators own their data and can
release it according to their own will.</p>


      
      <p class="MsoNormal" style="text-align: justify;">(3)
to be
simple to use either in the curation process or the query of the
database. For this purpose we are currently developing an advanced API
to insulate the user from the underlying data model and to provide
simple
methods for the user to deposit to or query from the database.</p>


      
      <p class="MsoNormal" style="text-align: justify;"><b><span style="font-size: 14pt;"></span></b></p>


      
      <p class="MsoNormal" style="text-align: justify;"><b><span style="font-size: 14pt;">Overview:</span></b></p>


      
      <p class="MsoNormal" style="text-align: justify;">The
PAZAR
system is currently developed as a mySQL database featuring a complex <a href="pazar_20051216.png" target="_blank">schema</a>
which allows for a high level of flexibility regarding the type of
information that can be captured.&nbsp;
The database <a href="pazar_dictionary.html" target="_blank">dictionary</a>
and an explanation of the <a href="iosys.htm" target="_blank">input/output
system</a> can help you find out some of the database constraints
and internal structure.&nbsp;</p>


      
      <p class="MsoNormal" style="text-align: justify;">To
ease the insertion of data into
the database, we are&nbsp; developing two curation interfaces, one
allowing the curator to capture higher levels of details than the
other. We have also designed an XML exchange format (<a href="http://www.w3schools.com/xml/default.asp" target="_blank">What's
XML?</a>) that can be used to
format already existing datasets. The <a href="pazar.dtd">PAZAR
Document Type Definition</a> (DTD) is now available as well as a <a href="PAZAR_DTD_DOC.pdf">documentation</a>
explaining its use.&nbsp; Some examples are also provided to help
the user understand the XML format:<br>


      </p>


      
      <ul>


        <li><a href="pazarexample.xml">example
1</a> describes an interaction between a transcription factor and
a binding
site located upstream a gene, and a set of mutations affecting this
interaction.</li>


        <li><a href="pazarexample2.xml">example
2</a>
describe a SELEX experiment with an heterodimer transcription factor
and the matrix built from the sequences.</li>


        <li><a href="pazarexample3.xml">example 3</a>
describes a gene reporter assay and the influence of co-expression with
a transcription factor.</li>


      
      </ul>


      
      <p class="MsoNormal" style="text-align: justify;"><b><span style="font-size: 14pt;"></span></b>As
an OPEN
SYSTEM, each boutique operator within PAZAR is welcome to participate
in further
API development and to create and maintain their own annotation
interfaces. Two forums are also available so that everyone can ask for
help (<a href="https://sourceforge.net/forum/forum.php?forum_id=520428" target="_blank">'Help' forum</a>) or make any comment
or suggestion (<a href="https://sourceforge.net/forum/forum.php?forum_id=512784" target="_blank">'Open Discussion' forum</a>).</p>


      
      <p class="MsoNormal" style="text-align: justify;"></p>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"><span style="font-weight: bold;"></span><b><b><span style="font-size: 14pt;">Project Members:</span></b></b></p>


Wyeth W. Wasserman, PI<br>


Jay R. Snoddy, PI <br>


Stefan Kirov, Postdoctoral fellow<br>


Elodie Portales-Casamar, Postdoctoral fellow <br>


Jonathan Lim, Software developer<b><br>


      </b> <br>


      
      <table style="text-align: left; width: 160px; height: 41px;" border="0" cellpadding="2" cellspacing="2">


        <tbody>


          <tr>


            <td><b><b><span style="font-size: 14pt;">Team e-mail:</span></b></b></td>


            <td><b><a href="mailto:pazar@cmmt.ubc.ca"><img style="border: 0px solid ; width: 29px; height: 21px;" alt="" src="email.gif"></a></b></td>


          </tr>


        
        </tbody>
      
      </table>


      
      <p class="MsoNormal" style="margin-left: 0.5in; text-indent: -0.5in;"></p>


      </td>


    </tr>


  
  </tbody>
</table>


</center>


</body>
</html>
