<h1>Describe your Cis Regulatory
Element</h1>

<form style="height: 546px;" action="http://serverpath/psite_get_cre.cgi" method="post" name="CRE">
  <br>
  <hr style="width: 100%; height: 2px;">
  <h2>Transcription factor/complex binding to this CRE (if known)</h2>
  <p>
    <input value="Add TF/TF complex" name="TFcomplexadd" onclick="open_cgi(CRE.userid.value,CRE.aname.value,'TFcomplex',file);" type="button">
  </p>
  <hr>
  <h2>CRE details</h2>
  <p>Gene/CRE description</p>
  <p>
    <textarea name="genedesc" cols="80" id="genedesc"></textarea>
  </p>
  <input name="mutset" value="no" type="hidden">
  Type 
  <p> 
    <input name="reg_type" onclick="CRE.end.disabled = true; CRE.sequence.disabled = false;  CRE.Add.disabled=true; CRE.mpos.disabled=true; CRE.mutantid.disabled=true;
 CRE.mutantdesc.disabled=true; CRE.constructname.disabled=true; CRE.constructdescr.disabled=true; CRE.mutbase.disabled = true; CRE.cseq.disabled=true; sequence.focus()" value="exact" checked="checked" type="radio">
    Exact 
    <input name="reg_type" value="region&quot;" onclick="CRE.end.disabled = false; CRE.sequence.disabled = true; CRE.Add.disabled=true; CRE.mpos.disabled=true; CRE.mutantid.disabled=true;
  CRE.mutantdesc.disabled=true; CRE.mutbase.disabled = true; CRE.constructname.disabled=true; CRE.constructdescr.disabled=true; CRE.cseq.disabled=true;" type="radio">
    Region 
    <input name="reg_type" value="point mutation" onclick=" CRE.mutbase.disabled = false; CRE.sequence.disabled = false; CRE.end.disabled = true; CRE.mpos.disabled=false; CRE.Add.disabled=false; CRE.mutantid.disabled=false;
 CRE.mutantdesc.disabled=false; CRE.constructname.disabled=true; CRE.constructdescr.disabled=true; CRE.cseq.disabled=true;" type="radio">
    Point mutation 
    <input type="radio" name="reg_type" value="construct" onclick=" CRE.end.disabled = true; CRE.sequence.disabled = false;  CRE.Add.disabled=true; CRE.mpos.disabled=true; CRE.mutantid.disabled=true;
 CRE.mutantdesc.disabled=true; CRE.mutbase.disabled = true; sequence.focus(); CRE.constructname.disabled=false; CRE.cseq.disabled=false; CRE.constructdescr.disabled=false;">
    Construct</p>
  <h3>CRE genome location data</h3>
  <h4>Enter here unique genome identification parameters in chromosome coordinates.</h4>
  <p>The chromosome coordinates should be correct for the current Ensembl release 
    and the respective genome build. The best way to set those is through the 
    'Get chromosome coordinates' button.</p>
  <p> start 
    <input name="start" type="text" id="start">
    end 
    <input name="end" type="text" id="end">
  </p>
  <p>chromosome 
    <input type="text" name="chromosome" id="chromosome">
  </p>
  <p>oranism 
    <input name="organism" type="text" id="organism">
    Genome Build 
    <input name="build" type="text" id="build">
  </p>
  <p>sequence (optional) 
    <input name="sequence" type="text" id="sequence">
  </p>
  <p>Ensembl representative transcript stable id (optional) 
    <input name="tid" type="text" id="tid">
  </p>
  <p>TSR definition start (optional) 
    <input name="fstart" type="text" id="fstart">
    TSR definition end (optional) 
    <input name="fend" type="text" id="fend">
  </p>
  <p> 
    <input name="chrcoord" type="button" id="chrcoord" value="Get Chromosome Coordinates" tooltiptext="Press to get the coordinates from the current Ensembl release" onMouseOver="return escape('test')" onclick="PopUp('http://htpath/chrcoordinates.htm')" >
  </p>
  <hr>
  <p>Construct name 
    <input name="constructname" type="text" disabled="true" size="32" maxlength="64">
    Construct Sequence 
    <input name="cseq" type="text" id="cseq" size="40" disabled="true">
  </p>
  <p>
    <textarea name="constructdescr" cols="80" id="constructdescr" disabled="true" ></textarea>
    Construct description </p>
<p> Do you want a check to be performed for conservation within this region (might be slow)?
 <input type="checkbox" name="Check for conservation" value="conscheck" checked
onclick="onoff(CRE.groupcons)";></p>
If yes what groups of species?
<select name="groupcons"  multiple>
<option value="mammals">Mammals</option>
<option value="insects">Insects</option>
<option value="fish">Fish</option></select>
  <p> (relative (to the
transcription start site, please type -35 for a sequence starting 35 bp
before the transcription start (TSS)site and 35 if the site is
downstream of the TSS, for example in the 5' UTR) </p>
  <p>&nbsp; </p>
  <p>mutation
position in element <input size="5" disabled="disabled" name="mpos"><br>
  </p>
  <p>mutant name <input disabled="disabled" name="mutantid"><br>
  </p>
Mutant
descritption<br>
  <p><textarea disabled="disabled" cols="80" rows="2" name="mutantdesc"></textarea></p>
  <p>mutated to
  <select name="mutbase" disabled="true"><option value="a">A</option><option value="c">C</option><option value="g">G</option><option value="t">T</option></select>
  </p>
  <input disabled="disabled" value="Add to set of mutations" name="Add" onclick="CRE.mpos.disabled=true; CRE.mutbase.disabled=true; open_cgi(CRE.userid.value,CRE.aname.value,'mutation',file);" type="button"><br>
  <br>
  <hr>
  <p> 
    <input name="submit" id="submit" value="Next" type="submit">
    <input name="cancel" id="cancel" value="Cancel" type="reset">
  </p>
  </form>
